docs/image_import.md

```diff
@@ -0,0 +1,42 @@
+# Image Import into Google Compute Engine (GCE)
+
+# Introduction
+We provide tooling and instructions for migrating your existing virtual
+machine images to GCE.
+
+Tools:
+* [Daisy](daisy.md)
+* [Import Precheck](../import_precheck/README.md)
```

Comment:
Change this line to

    * [Import Precheck](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/import_precheck/

or

    * [Import Precheck](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/import_precheck/README.md

Reply:
Done.